lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -1043,6 +1040,10 @@ MStatus MtohRenderOverride::Render(
             //Create a HydraViewportInformation 
             const Fvp::InformationInterface::ViewportInformation hydraViewportInformation(std::string(panelName.asChar()), cameraName);
             manager.AddViewportInformation(hydraViewportInformation, _renderIndexProxy, _lastFilteringSceneIndexBeforeCustomFiltering);
+            //Update the selection since we may have added scene indices through manager.AddViewportInformation to the merging scene index
```

Comment:
Shouldn't we do this then only when we add scene indices to the merging scene index?  Otherwise, we're setting the Hydra scene index selection at every render, which seems wasteful.

Reply:
You are right, I modified the code to only do that when we have added data producer scene indices